plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/featured-category/deprecated.tsx

```diff
@@ -25,12 +25,12 @@ const v1 = {
 	},
 	save: () => <InnerBlocks.Content />,
 	isEligible: ( attributes: BlockAttributes ) => {
-		// If the block has showDesc attribute as boolean value, it's a legacy block
+		// If the block has editMode attribute as boolean value, it's a legacy block
 		// and it should be migrated to use inner blocks instead.
-		return typeof attributes.showDesc === 'boolean';
+		return typeof attributes.editMode === 'boolean';
 	},
 	migrate: ( attributes: BlockAttributes, innerBlocks: BlockInstance[] ) => {
-		const { showDesc, ...otherAttributes } = attributes;
+		const { editMode, showDesc, ...otherAttributes } = attributes;
```

Comment:
`editMode` isn't used, here:

```suggestion
		const { showDesc, ...otherAttributes } = attributes;
```

Reply:
It is not used, but I need to destructure it there so that I exclude it from `...otherAttributes` which are then reassigned to the migrated block. If I don't do it, `otherAttributes` will contain `editMode` from the legacy block.

Reply:
True, I missed that!